api/k8s/v1/k8s.proto

```diff
@@ -288,6 +288,11 @@ message ListOptions {
 }
 
 message ListPodsRequest {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.k8s.v1.Pod",
+    pattern : "{cluster}/{namespace}"
+  };
```

Comment:
looking at this a second time i think i see why it wasn't added the first time around. there's some nuance here.

this annotation doesn't quite make sense, because it would say it's a single pod. but there's no ID, which is not a valid pod. so it might be confusing to look at pods that were touched in the audit log and have some with no id. i think it might work anyways but it's an open question.

i think what we might actually want here is something like `clutch.k8s.v1.Namespace` since that's what the user is acting on.

Reply:
But the pattern highlights "{cluster}/{namespace}" right? which would specify only the cluster and the namespace and the audit log would also specify the action which is `ListPods`